pandas/tests/groupby/test_groupby.py

```diff
@@ -1724,27 +1724,82 @@ def test_pivot_table_values_key_error():
         [0],
         [0.0],
         ["a"],
-        [Categorical([0])],
+        Categorical([0]),
         [to_datetime(0)],
-        [date_range(0, 1, 1, tz="US/Eastern")],
-        [pd.array([0], dtype="Int64")],
-        [pd.array([0], dtype="Float64")],
-        [pd.array([False], dtype="boolean")],
+        date_range(0, 1, 1, tz="US/Eastern"),
+        pd.array([0], dtype="Int64"),
+        pd.array([0], dtype="Float64"),
+        pd.array([False], dtype="boolean"),
     ],
 )
 @pytest.mark.parametrize("method", ["attr", "agg", "apply"])
 @pytest.mark.parametrize(
     "op", ["idxmax", "idxmin", "mad", "min", "max", "sum", "prod", "skew"]
 )
-def test_empty_groupby(columns, keys, values, method, op):
+def test_empty_groupby(columns, keys, values, method, op, request):
     # GH8093 & GH26411
 
+    if isinstance(values, Categorical) and len(keys) == 1 and method == "apply":
```

Comment:
ok, plan is to address the root cause and unxfail these as appropriate at some point?

Reply:
yes, immediately xref #41342

Reply:
(will need to rebase #41342 after this)